githooks/prepare-commit-msg

```diff
@@ -96,6 +96,23 @@ ${cchar}Release note (bug fix): \\
 "
 fi
 
+# Add an issue or epic reference.
+if ! grep -q -E '^(?i:close[sd]?|fix(?:e[sd])?|resolve[sd]?)' "$1"; then
+	sed_script+="/$cchar Please enter the commit message for your changes./i\\
+$cchar Please enter a valid issue or epic reference:\\
+${cchar}Fixes: #77376\\
+${cchar}Part of https://cockroachlabs.atlassian.net/browse/CRDB-13558\\
+${cchar}Informs: https://github.com/cockroachdb/cockroach/issues/33316\\
+${cchar}Epic: CRDB-8035\\
+${cchar}Epic: none\\
```

Comment:
Let's put the `Epic: none` example first in the list. It's what people will copy/paste most often.